index.html

```diff
@@ -0,0 +1,83 @@
+                </div>
+            </header>
+            <main class="content">
+                <div class="info">
+                    <div class="about">
+                        O mnie
+                    </div>
+                    <div class="about-text">
+                        Jestem pociesznym zwierzątkiem z bogatym doświadczeniem w leczeniu ludzkich smutków. Szukam dobrego domu, takiego na zawsze.
+                    </div>
+                    <hr class="line-short">
+                    <div class="interest">Zainteresowania</div>
+                    <div class="interest-text"
+                        <ul>
+                            <li>jedzenie warzywek</li>
+                            <li>leżenie na sianku</li>
+                            <li>popcorningowanie</li>
+                            <li>patrzenie w przestrzeń</li>
+                        </ul>
+                    </div>
+                    <hr class="line-short">
+                    <div class="additional">Dodatkowe informacje</div>
+                    <div class="additional-text">
+                        <ul>
+                            <li>Wolontariat w Stowarzyszeniu Pomocy Świnkom Morskim</li>
+                            <li>Występ gościnny na kanale "Geografia ze świnkami" w odcinku</li>
+                            <a href="https://www.youtube.com/watch?v=TPtu6H7ZK6w">Sealand - najmniejsze i najdziwniejsze "państwo" świata</a>
+                        </ul>
+                    </div>
+                </div>
+                <div class="qualifications">
+                    <div class="education">Wykształcenie</div>
+                    <div class="university">
+                        Uniwersytet SWPS w Sopocie - Psychologia; specjalizacja: kawioterapia.
+                        Rok ukończenia: 2023.
+                    </div>
+                    <hr class="line-long">
+                    <div class="experience">Doświadczenie zawodowe</div>
+                    <div class="experience-text">
+                        <div>02.2024 - obecnie - Stowarzyszenie Pomocy Świnkom Morskim - Poszukiwacz domu stałego</div>
+                        <br>
+                        <div>01.2023-12.2023 - Dom Seniora Milorstowo - Specjalista ds. kawioterapii</div>
+                        <br>
+                        <div>06.2022-12.2022 - Przychodnia Weterynaryjna "Morska" - Stażysta</div>
+                    </div>
+                    <hr class="line-long">
+                    <div class="certificates">Certyfikaty i kursy</div>
+                    <div class=certificates-text>
+                        <div>Certyfikat "Świnka przyjaciel", poziom I oraz II</div>
+                        <br>
+                        <div>Kurs "Jaki ładny chomik - jak grzecznie odpowiadać"</div>
+                    </div>
+                </div>
+            </main>
+            <footer>
+                <div class="disclaimer">
```

Comment:
You could remove this `<div>` and add the `disclaimer` class to the footer instead